extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -45,7 +45,7 @@ export class BookTocManager implements IBookTocManager {
 		this.tableofContents = [];
 		this._movedFiles = new Map<string, string>();
 		this._modifiedDirectory = new Set<string>();
-		this._tocFiles = new Map<string, JupyterBookSection[]>();
+		this._tocFiles = new Map<string, string>();
```

Comment:
You can just instantiate these all where they're defined - no need to do that here